src/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -0,0 +1,71 @@
+import d3 from 'd3';
+import $ from 'jquery';
+import { VislibVisualizationsChartProvider } from './_chart';
+import { GaugeTypesProvider } from './gauges/gauge_types';
+
+export function GaugeChartProvider(Private) {
+
+  const Chart = Private(VislibVisualizationsChartProvider);
+  const gaugeTypes = Private(GaugeTypesProvider);
+
+  class GaugeChart extends Chart {
```

Comment:
One of the visualizations is "higher" than the other. It almost looks like one is center aligned, and the other one is bottom aligned.